### PR TITLE
Run dotnet format on solution

### PR DESCRIPTION
**Pull request by cadatoiva**
This pull request is the result of running `dotnet format` with our current style guidelines. [According to their wiki](https://github.com/dotnet/format/wiki/Supported-.editorconfig-options), and basically only changes whitespace and file encoding type. Some files were originally UTF-8-BOM which is a deprecated encoding, and have been brought in to the standard UTF-8 encoding.

### CODE REVIEW

File: src/Jackett.Common/ExceptionWithConfigData.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```
Comment (cadatoiva) [inline src/Jackett.Common/ExceptionWithConfigData.cs]: Files with a change like this on line 1 are the ones that had their encoding changed to utf-8.

Several files have only this change.

File: src/Jackett.Common/Indexers/Abstract/AvistazTracker.cs [hunk 2]
```diff
@@ -112,12 +112,14 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
...
-                    var qLink = qRow.Find("a.torrent-filename"); ;
+                    var qLink = qRow.Find("a.torrent-filename");
+                    ;
                     release.Title = qLink.Text().Trim();
                     release.Comments = new Uri(qLink.Attr("href"));
                     release.Guid = release.Comments;
 
-                    var qDownload = qRow.Find("a.torrent-download-icon"); ;
+                    var qDownload = qRow.Find("a.torrent-download-icon");
+                    ;
```
Comment (ngosang) [inline src/Jackett.Common/Indexers/Abstract/AvistazTracker.cs]: fix manually

File: src/Jackett.Common/Indexers/Abstract/AvistazTracker.cs [hunk 3]
```diff
@@ -112,12 +112,14 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
                     release.MinimumRatio = 1;
                     release.MinimumSeedTime = 172800; // 48 hours
 
-                    var qLink = qRow.Find("a.torrent-filename"); ;
+                    var qLink = qRow.Find("a.torrent-filename");
+                    ;
```
Comment (ngosang) [inline src/Jackett.Common/Indexers/Abstract/AvistazTracker.cs]: fix manually

File: src/Jackett.Common/Indexers/CardigannIndexer.cs [hunk 4]
```diff
@@ -135,7 +135,8 @@ public CardigannIndexer(IIndexerConfigurationService configService, Utils.Client
                 }
                 else
                 {
-                    item = new StringItem { Value = Setting.Default }; ;
+                    item = new StringItem { Value = Setting.Default };
+                    ;
```
Comment (ngosang) [inline src/Jackett.Common/Indexers/CardigannIndexer.cs]: fix manually

File: src/Jackett.Common/Indexers/Fuzer.cs [hunk 5]
```diff
@@ -329,7 +329,8 @@ private async Task<string> getHebName(string searchTerm)
                 if (link.Text().Trim().ToLower() == searchTerm.Trim().ToLower())
                 {
                     var address = link.Attr("href");
-                    if (string.IsNullOrEmpty(address)) { continue; }
+                    if (string.IsNullOrEmpty(address))
+                    { continue; }
```
Comment (ngosang) [inline src/Jackett.Common/Indexers/Fuzer.cs]: fix manually

File: src/Jackett.Common/Services/ConfigurationService.cs [hunk 6]
```diff
@@ -175,7 +175,8 @@ public string ApplicationFolder()
         public string GetContentFolder()
         {
             // If we are debugging we can use the non copied content.
-            string dir = Path.Combine(ApplicationFolder(), "Content"); ;
+            string dir = Path.Combine(ApplicationFolder(), "Content");
+            ;
```
Comment (ngosang) [inline src/Jackett.Common/Services/ConfigurationService.cs]: fix manually

File: src/Jackett.Common/Services/ConfigurationService.cs [hunk 7]
```diff
@@ -204,7 +205,8 @@ public List<string> GetCardigannDefinitionsFolders()
             }
 
             // If we are debugging we can use the non copied definitions.
-            string dir = Path.Combine(ApplicationFolder(), "Definitions"); ;
+            string dir = Path.Combine(ApplicationFolder(), "Definitions");
+            ;
```
Comment (ngosang) [inline src/Jackett.Common/Services/ConfigurationService.cs]: fix manually

File: src/Jackett.Common/Services/ConfigurationService.cs [hunk 8]
```diff
@@ -266,7 +268,8 @@ public ServerConfig BuildServerConfig(RuntimeSettings runtimeSettings)
 
                 // Check for legacy settings
 
-                var path = Path.Combine(GetAppDataFolder(), "config.json"); ;
+                var path = Path.Combine(GetAppDataFolder(), "config.json");
+                ;
```
Comment (ngosang) [inline src/Jackett.Common/Services/ConfigurationService.cs]: fix manually

File: src/Jackett.Common/Utils/ParseUtil.cs [hunk 9]
```diff
@@ -20,7 +20,8 @@ public static string NormalizeSpace(string s)
 
         public static string NormalizeMultiSpaces(string s)
         {
-            return new Regex(@"\s+").Replace(NormalizeSpace(s), " "); ;
+            return new Regex(@"\s+").Replace(NormalizeSpace(s), " ");
+            ;
```
Comment (ngosang) [inline src/Jackett.Common/Utils/ParseUtil.cs]: fix manually

File: src/Jackett.Server/Controllers/UIController.cs [hunk 10]
```diff
@@ -44,7 +44,8 @@ public async Task<IActionResult> Login()
                 return Redirect("Dashboard");
             }
 
-            return new PhysicalFileResult(config.GetContentFolder() + "/login.html", "text/html"); ;
+            return new PhysicalFileResult(config.GetContentFolder() + "/login.html", "text/html");
+            ;
```
Comment (ngosang) [inline src/Jackett.Server/Controllers/UIController.cs]: fix manually

**Comment by ngosang** [comment]
@garfield69 merge this PR before the others. If not the rebase will be painful.

File: src/Jackett.Common/Indexers/Abstract/AvistazTracker.cs (hunk 2 above)
Comment (garfield69) [reply src/Jackett.Common/Indexers/Abstract/AvistazTracker.cs]: @ngosang you have marked your review as approved, but have these "fix manually" comments... so presumably you want changes? And I should hold back merging?

**Comment by cadatoiva** [comment]
Since we're doing the refactoring piecemeal, there's going to be artifacts like the above that don't get caught the first couple times around. The automatic tools will get around to removing things like redundant semi-colons, braces, parenthesis usings etc.

It is ironic how many of the double semi-colons have shown up though.

File: src/Jackett.Common/Indexers/Abstract/AvistazTracker.cs (hunk 2 above)
Comment (ngosang) [reply src/Jackett.Common/Indexers/Abstract/AvistazTracker.cs]: Sorry for the confusion. This PR can be merged, but there are little details that I want to be fixed. @cadatoiva are you going to do the changes in this PR?

Comment (cadatoiva) [reply src/Jackett.Common/Indexers/Abstract/AvistazTracker.cs]: No, they'll fix themselves in a future PR. I'm just going through each category of automated fixes right now. We'll keep note of the details and make sure they're fixed in the future.

Comment (cadatoiva) [reply src/Jackett.Common/Indexers/Abstract/AvistazTracker.cs]: The scope of this PR is whitespace and encoding issues.

Comment (ngosang) [reply src/Jackett.Common/Indexers/Abstract/AvistazTracker.cs]: @garfield69 you can merge

**Pull request merged**